src/Client/Log.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace HelloFresh\Stats\Client;
+
+
+use HelloFresh\Stats\Client;
+use HelloFresh\Stats\Incrementer;
+use HelloFresh\Stats\State;
+use HelloFresh\Stats\Timer;
+use Psr\Log\LoggerInterface;
+
+class Log extends Base implements Client
+{
+    /** @var LoggerInterface */
+    protected $logger;
+
+    /**
+     * Log constructor.
+     *
+     * @param LoggerInterface $logger
+     */
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+        $this->resetHTTPRequestSection();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function buildTimer()
+    {
+        return new Timer\Memory();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function getIncrementer()
+    {
+        return new Incrementer\Log($this->logger);
```

Comment:
Should we not store the `Incrementer\Log` instance for later use? (same question for [line 49](https://github.com/hellofresh/stats-php/pull/1/files#diff-fb64cec0d1d5379aeae9d01b5fee5c4dR49) and the [NoOp](https://github.com/hellofresh/stats-php/pull/1/files#diff-ef31bdea77aabeb99a212c370f814750) version)